### PR TITLE
[ES] Add `Ranking criteria`

### PR DESCRIPTION
**Pull request by agatemosu**
por fin

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

File: wiki/Ranking_criteria/es.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+- **Si el título de una canción contiene marcadores de versión corta o de juego, los marcadores deben estandarizarse a `(Short Ver.)` y `(Game Ver.)` respectivamente.**
+- **Las versiones cortadas no oficiales de las canciones deben añadir un marcador `(Cut Ver.)` al final del título actual.** Si ya hay un marcador de duración en el título de la canción, `(Cut Ver.)` lo sustituirá. Esto se hace para distinguir los cortes no oficiales de una canción de las versiones completas. No se considerarán cortadas las canciones que se hayan acortado de forma que casi coincidan con sus versiones oficiales, ni las canciones que sean un bucle completo de una pista en bucle.
+  - Nota: Si un corte no oficial contiene secciones coincidentes en el mismo orden y tiene aproximadamente la misma duración que el tamaño oficial para televisión, una versión corta o una versión para juegos, contará como corte oficial y se usará el marcador correspondiente. Las covers y remezclas no cuentan.
```
[the remark this answers is not in the document]
Comment (agatemosu) [reply wiki/Ranking_criteria/es.md:111]: ya es correcto, cover viene de cubrir y el término correcto es "versión". los dos están en femenino

File: wiki/Ranking_criteria/es.md [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+- **El primer [punto de tiempo no heredado](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point) de un beatmap no puede usarse para alternar el [kiai](/wiki/Gameplay/Kiai_time).** Hacer esto hará que el kiai parpadee antes de que aparezcan los objetos. En su lugar, debe usarse un [punto heredado](/wiki/Client/Beatmap_editor/Timing#inherited-timing-point) en la misma posición que el primer punto no heredado para alternar el kiai.
+- **Los objetos deben encajar com menos de 2 ms de cualquier marca de la línea de tiempo.** [AiMod](/wiki/Client/Beatmap_editor/AiMod) informará de estos problemas, así como de los raros falsos positivos. Los falsos positivos se producen principalmente en los extremos de los sliders y en los repetidores, y deben verificarse manualmente o con otras herramientas. Los objetos de una sección musical que requieran divisores de tiempo no compatibles (p. ej. 1/11) pueden:
```
Comment (MChecaH) [inline wiki/Ranking_criteria/es.md:185]:
```suggestion
- **Los objetos deben encajar con menos de 2 ms de cualquier marca de la línea de tiempo.** [AiMod](/wiki/Client/Beatmap_editor/AiMod) informará de estos problemas, así como de los raros falsos positivos. Los falsos positivos se producen principalmente en los extremos de los sliders y en los repetidores, y deben verificarse manualmente o con otras herramientas. Los objetos de una sección musical que requieran divisores de tiempo no compatibles (p. ej. 1/11) pueden:
```

File: wiki/Ranking_criteria/es.md [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+- **Cada archivo de hitsound debe tener una duración mínima de 25 ms.** Los archivos más cortos pueden hacer que no se reproduzca ningún sonido en el juego.
+- **Los archivos de sonido completamente en silencio deben usar [este archivo de 44 bytes](https://up.ppy.sh/files/blank.wav).** Otros archivos tienen tamaños innecesariamente grandes y los archivos de 0 bytes no funcionan.
+- **Los [hitsounds del storyboard](/wiki/Beatmapping/Hitsound#storyboarded-hitsound) no pueden ser usados como reemplazo de los [hitsounds activos](/wiki/Beatmapping/Hitsound#active-hitsound).** Estos proporcionan un feedback inexacto al jugador. Los hitsounds del storyboard en otras situaciones son aceptables, pero se desaconsejan. osu!mania está exento de esta norma.
+
+### Pautas
+
+- **El archivo de audio y los archivos de hitsounds de un beatmap no deben presentar distorsiones de sonido audibles e injustificadas**, como recortes, silenciamientos o crujidos que claramente no han sido intencionados por el artista o ni forman parte de la identidad de la canción. Esto se determina mejor escuchando el audio que usando un programa informático.
```
Comment (MChecaH) [inline wiki/Ranking_criteria/es.md:214]: `ni` por si solo ya actua como conjunción. El `o` es redundante en tal caso.
```suggestion
- **El archivo de audio y los archivos de hitsounds de un beatmap no deben presentar distorsiones de sonido audibles e injustificadas**, como recortes, silenciamientos o crujidos que claramente no han sido intencionados por el artista ni forman parte de la identidad de la canción. Esto se determina mejor escuchando el audio que usando un programa informático.
```

**Review by MChecaH — CHANGES_REQUESTED** [review]
Quick, tiny comments to be made to grammar and ensuring it stays true to the original article.

**Comment by agatemosu** [comment]
i forgot sorry i'm looking at the rest

**Review by TicClick — APPROVED** [review]

**Comment by TicClick** [comment]
(approved to get this merged in sooner rather wait another period of time for the PR from https://github.com/ppy/osu-wiki/pull/10320#issuecomment-1784162379 to become ready)

**Comment by agatemosu** [comment]
> (approved to get this merged in sooner rather wait another period of time for the PR from [#10320 (comment)](https://github.com/ppy/osu-wiki/pull/10320#issuecomment-1784162379) to become ready)

OK so the unknownosu review will be in other pr

**Pull request merged**